Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -446,6 +446,13 @@ define([
     var scratchCartesian = new Cartesian3();
     var scratchCartesianTwo = new Cartesian3();
     var scratchCartesianThree = new Cartesian3();
+    var scratchZoomViewOptions = {
+      orientation: {
```

Comment:
Can we use [HeadingPitchRoll](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/HeadingPitchRoll.js) here?